account_invoice_import/wizard/account_invoice_import.py

```diff
@@ -487,19 +481,49 @@ def invoice_already_exists(self, commercial_partner, parsed_inv):
             ], limit=1)
         return existing_inv
 
+    def _search_import_config(self, partner, company_id):
+        return self.env['account.invoice.import.config'].search([
+            ('partner_id', '=', partner.id),
+            ('company_id', '=', company_id)])
+
+    def _get_missing_config(self, partner, company):
+        configs = partner.invoice_import_ids.filtered(
+            lambda r: r.company_id == company)
+        if not configs:
+            raise UserError(_(
+                "Missing Invoice Import Configuration on partner '%s'."
+            ) % partner.display_name)
+        return configs[0].convert_to_import_config()
+
+    def _get_company(self, company_id, parsed_inv):
+        """This hook allows us to define ways to set/check the company"""
```

Comment:
I found this method confusing because there already is a company_id and it restricts the candidates for this method. Maybe rename to `_get_child_company`?